.github/workflows/build.yml

```diff
@@ -46,7 +46,7 @@ jobs:
           echo $installed
           [[ $installed =~ "Python ${expected}" ]] && echo "Configured Python" || (echo "Failed to configure Python" && exit 1)
       - name: Start algod
-        run: make algod-start
+        run: make algod-start-report
```

Comment:
adding a printout of the commit-hash when C.I. stands up the algod.

Reply:
you want to know the version code to locate some dryrun related behavior difference?

Reply:
Yes. That's how I was able to understand that latest graviton test failures were because I was running an old version of algod. In fact, I ended up needing to run `docker compose pull` to fix the discrepancy.